### ISSUE TITLE
List of Domains is empty

### ISSUE THREAD

**Issue by MarkChr** [RainLoop/rainloop-webmail#1109]
I have installed the standard Version (1.10.2.143) of Rainloop on my provider Webspace. Everything works fine, but after i have integrated 4 Email-Domains,  i can't see in the admin panel and can't edit them. The List of Domains are empty, but they work! When i look with FTP Client on my webspace all Email-Domains are there and when i login with my user i can see all domains, but can only change the order and change password.

![userpanel](https://cloud.githubusercontent.com/assets/20473393/16869303/74e90b74-4a7c-11e6-80fc-813c7cdf98df.JPG)
![adminpanel](https://cloud.githubusercontent.com/assets/20473393/16869390/ceabfc0c-4a7c-11e6-9fd9-4ce2f7340976.JPG)

Hope you understand me ... :-)

Regards  Mark

**Comment by jarlave** [comment]
just want to say that I have the same problem, everything works, but the domains aren't showing up.

**Comment by MarkChr** [comment]
UPDATE: today i've installed a fresh version of Rainloop (1.10.3.151) and when i've added a domain it was not listed or shown.

**Comment by vizv** [comment]
Since nobody gives out an useful log for debugging, here is one.

```
2016/09/19 08:07:16 [error] 15#0: *1028 FastCGI sent in stderr: "PHP message: PHP Notice:  Use of undefined constant GLOB_BRACE - assumed 'GLOB_BRACE' in /rainloop/rainloop/v/1.10.3.151/app/libraries/RainLoop/Providers/Domain/DefaultDomain.php on line 329
PHP message: PHP Warning:  glob() expects parameter 2 to be integer, string given in /rainloop/rainloop/v/1.10.3.151/app/libraries/RainLoop/Providers/Domain/DefaultDomain.php on line 329
PHP message: PHP Warning:  Invalid argument supplied for foreach() in /rainloop/rainloop/v/1.10.3.151/app/libraries/RainLoop/Providers/Domain/DefaultDomain.php on line 331" while reading response header from upstream, client: 10.0.0.5, server: , request: "POST /?/Ajax/&q[]=/0/ HTTP/1.0", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "mail-web", referrer: "https://mail.vizv.com/?admin"
```

**Comment by vizv** [comment]
So checked the code, this problem is caused by some OS doesn't support `GLOB_BRACE`, refer to https://github.com/zendframework/zend-stdlib/issues/58#issue-147689568, and http://php.net/manual/en/function.glob.php#refsect1-function.glob-notes.

Was introduced in 28a125426ac5340c81b4fe47d4e2cf4d3770fae4, and has already been fixed in 17669b7be0f852dc111109d1fcd52e2796cbd502.

For 1.10.3.151, you can manually patch `rainloop/v/1.10.3.151/app/libraries/RainLoop/Providers/Domain/DefaultDomain.php` as https://github.com/RainLoop/rainloop-webmail/blob/master/rainloop/v/0.0.0/app/libraries/RainLoop/Providers/Domain/DefaultDomain.php#L329-L330, which should fix the problem.

**Comment by MarkChr** [comment]
It works for me - Thanks a lot!
